EIPS/eip-2767.md

```diff
@@ -0,0 +1,239 @@
+---
+eip: 2767
+title: Contract Ownership Governance
+author: Soham Zemse (@zemse), Nick Mudge (@mudgen)
+discussions-to: https://github.com/ethereum/EIPs/issues/2766
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-07-04
+requires: 173, 165, 191
```

Comment:
Note: Two of these are still draft, and they'll need to go to final before this goes to final.

Reply:
Good catch.